extensions/amp-story/0.1/amp-story.js

```diff
@@ -478,6 +478,16 @@ export class AmpStory extends AMP.BaseElement {
       this.onKeyDown_(e);
     }, true);
 
+    // keep track of page-id in browser history API
```

Comment:
Nit: Comments start with an uppercase letter and end with a dot

Reply:
this comment went away in refactor